### ISSUE TITLE
null values cast to string on find, create and update

### ISSUE THREAD

**Issue by anthonmanix** [citycide/trilogy#79]
When selecting, inserting or updating the database, null values are cast to string. Is there an option to disable this functionality.

**Comment by citycide** [comment]
Hey @anthonmanix - I'm looking into it. In the meantime make sure you're on the latest version and all the other issue template goodness.

<details>
<summary>issue template</summary>

## BEFORE YOU SUBMIT

* read the [Contributor Guide](https://github.com/citycide/trilogy/blob/master/contributing.md)
* make sure the issue isn't already [being tracked](https://github.com/citycide/trilogy/issues)
* should be reproducible with the latest version of trilogy
	* ![](https://img.shields.io/npm/v/trilogy.svg)

You should always include as much information as possible about the problem -
help maintainers and contributors help you back:

* relevant error messages ( including stack traces )
* any uncommon or "abnormal" environment configurations
* link to a [minimal reproducible][mcve] case, ie. a GitHub repo
  ( this is the fastest way to a solution! )

If a section below isn't relevant feel free to get rid of it, but be
prepared to provide more information if necessary.

*Remember to delete the above section and the instructions in the sections below before submitting*

## description

> If this is a feature request, make a convincing case. Specific uses
  are best - what problem(s) does it solve for users?

> For bug reports, please provide _as much relevant information as possible_.

### source

```js
// try to avoid posting hundreds of lines of code
// edit it down to just the relevant parts
```

### error

```
COPY THE ERROR MESSAGE, INCLUDING STACK TRACE HERE
```

## expected

> What should happen in this situation?

## actual

> What happened that's considered unexpected?

[mcve]: https://stackoverflow.com/help/mcve
</details>

**Comment by citycide** [comment]
@anthonmanix released in [v1.4.2](https://github.com/citycide/trilogy/releases/tag/v1.4.2)

Relevant new tests [are here](https://github.com/citycide/trilogy/blob/3a5e6c5992c48ab29981cf6d34f5b6800c2b48d3/tests/create.js#L45-L67). Put `null` in, get `null` out - unless you've set `notNullable` in which case you get a better error now than before.
